code/game/machinery/launch_pad.dm

```diff
@@ -56,6 +56,9 @@
 	if(teleporting)
 		to_chat(user, "<span class='warning'>ERROR: Launchpad busy.</span>")
 		return
+	if(is_centcom_level(loc.z))
```

Comment:
I recommend getting the Z of get_turf(src), otherwise you're just relying on no-one ever adding a deep frier or a package wrapper on the syndi station